Source/Core/VideoBackends/D3D12/D3DBase.cpp

```diff
@@ -765,8 +765,10 @@ void Reset()
   s_xres = client.right - client.left;
   s_yres = client.bottom - client.top;
 
+  DXGI_SWAP_CHAIN_DESC desc = {};
+  s_swap_chain->GetDesc(&desc);
```

Comment:
The swap chain format and flags should be consistent here, so no need to use GetDesc(), just pass DXGI_FORMAT_R8G8B8A8_UNORM, DXGI_SWAP_CHAIN_FLAG_ALLOW_TEARING directly.